cmd/open.go

```diff
@@ -6,11 +6,33 @@
 
 package cmd
 
-import "github.com/tsuru/tsuru/exec"
+import (
+
+	"golang.org/x/sys/unix"
+	"github.com/tsuru/tsuru/exec"
+)
+
+func isWSL() bool {
+	var u unix.Utsname
+	err := unix.Uname(&u)
+	if err != nil {
+		fmt.Println(err)
+		return false
+	}
+	release := strings.ToLower(string(u.Release[:]))
+	return strings.Contains(release, "microsoft")
+}
 
 func open(url string) error {
+
+	cmd := "xdg-open"
+
+	if(isWSL()){
+		cmd = "powershell.exe -c start"
```

Comment:
@markiing, I supposed that need to separate cmd in args,
example
```
cmd = "powershell.exe"
args = []string{"-c", "start", url"}
```

Reply:
Done :D